lib/event_sourcery/postgres/optimised_event_poll_waiter.rb

```diff
@@ -14,34 +14,30 @@ def initialize(pg_connection:, timeout: 30, after_listen: proc { })
       def poll(after_listen: proc { }, &block)
```

Comment:
I think I prefer passing in the block and calling `block.call` as it's more explicit.